### PR TITLE
Add example successful documentation bugs

### LINKED ISSUE #118: add a new section in documentation chapter
In the introduction, you mention reporting bugs in documentation and actually this is probably the main way that people contribute. It might be helpful to expand on what is a bug in documentation - perhaps with a couple of examples of reporting a typo or reporting something that is no longer correct. If an R Core developer agrees there is a bug, they will typically fix this directly, so there is no need for anyone to contribute a patch.

For larger issues, people can contribute by raising larger issues with documentation (unclear or incomplete documentation), discussing how these issues might be addressed and potentially contributing a patch if there is support from R Core. I think the comment about R Core not necessarily accepting patches might go here - although I agree the introduction should not give the impression that any and all documentation contributions will be welcome.

So this should all come before discussing how to find documentation issues on Bugzilla. Taking Nicolas' point about the metadata, I think it is still helpful for people to know how to browse documentation issues without needing a directly link. So perhaps something like "Select "Browse" in the navigation bar, select the "R" category and then select the "Documentation" component ([direct link to Documentation bugs]())".

_Originally posted by @hturner in https://github.com/r-devel/rdevguide/pull/66#discussion_r982067403_

### PR DESCRIPTION
**Pull request by cjrace**
## Overview

Building on from the work in #183, I've copied across the changes started by @SJaffa on the RSECon24 R dev day into my own fork and then continued the work.

This resolves #118.

## Changes in this PR on top of what was in #183 

Added two examples of successfully closed bugs.

Removed duplicate paragraph highlighted by @llrs.

Added a line on the introduction page to highlight that documetation is often a good starting point.

### CODE REVIEW

File: 01-Introduction.Rmd
Comment (hturner) [inline 01-Introduction.Rmd:11]: Add/update links:
```suggestion
1. Contributing to bug fixing: Refer to [Issue Tracking](#IssueTrack) and [Reviewing Bugs](#ReviewBugs).
2. Contributing to translations: Refer to [Translations](#message-translations).
3. Testing R before release: Refer to [Testing Pre-release R Versions](#TestRVer).
```

File: 08-documenting.Rmd
Comment (hturner) [inline 08-documenting.Rmd:24]: This belongs elsewhere (as you suggest in #184), since the CRAN website is not part of (base) R's documentation, so we can delete this. 
```suggestion
```

File: 08-documenting.Rmd
Comment (hturner) [inline 08-documenting.Rmd:26]: This also applies to documentation issues in the base R packages, or other packages maintained by the R Core Team. Here we can link to https://contributor.r-project.org/rdevguide/ReportingBugs.html#ReportBug.

File: 08-documenting.Rmd
Comment (hturner) [inline 08-documenting.Rmd:28]: Need to differentiate between packages maintained by R core and contributed packages. See https://contributor.r-project.org/rdevguide/ReportingBugs.html#ReportBug.

File: 08-documenting.Rmd
Comment (hturner) [inline 08-documenting.Rmd:196]:
```suggestion
Contributions to the R Development Guide (what you are reading now) are also welcome. The source for this guide lives in a [GitHub repository](https://github.com/r-devel/rdevguide/) and bug reports should be submitted to the [issue tracker](https://github.com/r-devel/rdevguide/issues).
```

File: 08-documenting.Rmd
Comment (hturner) [inline 08-documenting.Rmd:198]: Either briefly explain what a pull request is, or briefly explain that people can also propose changes to the source directly as described in the linked section.

**Review by hturner — CHANGES_REQUESTED** [review]
Thanks for building on @SJaffa's PR. I have suggested various minor changes, some of which are updates to content that has been moved around - although you/Sarah didn't write this it would be good to make these improvements as part of this PR.

File: 01-Introduction.Rmd
Comment (cjrace) [reply 01-Introduction.Rmd]: Have moved back down to the bottom and specified that it's this guides code of conduct

File: 08-documenting.Rmd
Comment (cjrace) [reply 08-documenting.Rmd]: I've added the link to reporting bugs

File: 08-documenting.Rmd
Comment (cjrace) [reply 08-documenting.Rmd]: Have tried to make this section a bit clearer

File: 08-documenting.Rmd
Comment (cjrace) [reply 08-documenting.Rmd]: Shuffled parts around, pointing to the introduction section first, then adding a bit of context about PRs before the final paragraph about CI - how does that read now?

File: 09-documenting.Rmd
Comment (hturner) [inline 09-documenting.Rmd:24]: On second review, it would actually be better to link to the relevant sub-section:
```suggestion
To suggest improvements to the R manuals, report typos or bugs in the R manuals, or to raise issues related to documentation in packages maintained by the R Core Team, follow the usual guidance for [reporting bugs in R](#RCorePkgBug).
```

File: 09-documenting.Rmd
Comment (hturner) [inline 09-documenting.Rmd:26]: If the package has an issue tracker or specific way to report bugs that should be used rather than contacting the maintainer directly - this is all document in the reporting bugs section so we can refer to that.
```suggestion
To report bugs or typos in the documentation of a package that is not maintained by the R Core Team, follow the guidance for [reporting bugs in contributed packages](#nonRCorePkgBug).
```

File: 09-documenting.Rmd
Comment (hturner) [inline 09-documenting.Rmd:197]:
```suggestion
Our workflow for the R Development Guide uses continuous integration and deployment so changes to the live R Development Guide site are normally published once a pull request is merged. 
```

File: 08-documenting.Rmd
Comment (hturner) [reply 08-documenting.Rmd]: Looks good - just suggested a minor change to be consistent about the name of the guide.

File: 08-documenting.Rmd
Comment (hturner) [reply 08-documenting.Rmd]: Thanks, made a couple of suggestions to finalise this part.

**Review by hturner — COMMENTED** [review]
Suggesting a few minor revisions before we merge in.

**Pull request merged**
